turbo.json

```diff
@@ -1,5 +1,8 @@
 {
   "pipeline": {
+    "db-generate": {
+      "outputs": [".generated/**"]
+    },
```

Comment:
I'm curious, why bother hoisting `db-generate` to the top here?

Reply:
I like having this read "in the order things happen", the other order felt arbitrary